### DISCUSSION TITLE
Disable light/dark mode switching

### DISCUSSION THREAD

**Discussion by scott-huberty** [lepture/shibuya#42] [Q&A] [answered]
With the Shibuya theme, is it possible to disable the light/dark switch icon, and force the documentation to be shown in light mode, for example? If so, how can I do this?

**Comment by lepture** [comment, marked as answer]
With the latest release, you can create an empty `components/theme-switch.html` in your docs `_templates` folder.

**Reply by scott-huberty** [reply]
Thanks, @lepture !
